operator/v1/types_kubeapiserver.go

```diff
@@ -35,6 +35,23 @@ type KubeAPIServer struct {
 
 type KubeAPIServerSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+
+	// eventTTL specifies the amount of time that the events are stored before being deleted.
+	//
+	// The value must be parseable as a time duration value;
+	// see <https://pkg.go.dev/time#ParseDuration>.
+	//
+	// If configured, it must be a value of 1m (one minute) or greater, we only allow setting
+	// minute and hour durations (e.g. 5m or 5h).
+	//
+	// The default value is 3h.
+	//
+	// +kubebuilder:validation:Format=duration
+	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(m|h))+)$
+	// +kubebuilder:validation:Type:=string
+	// +default="3h"
```

Comment:
Since this is a configuration API, we want to reserve the right to change this value over time. This will be easier if we don't set the value through openapi defaulting. 

It's better here to set the value in the controller by detecting that it is omitted.

We generally add a comment to the godoc to explain this scenario

```
// When omitted this means no opinion, and the platform is left to choose a reasonable default, which is subject to change over time.
// The current default value is 3h.
```